c2rust-analyze/src/util.rs

```diff
@@ -85,6 +85,16 @@ pub enum Callee<'tcx> {
     },
     /// A built-in or standard library function that requires no special handling.
     MiscBuiltin,
+    /// libc::malloc
+    Malloc,
+    /// libc::calloc
+    Calloc,
```

Comment:
Eventually, I think we'd want to just include everything with `__attribute__ ((__malloc__))`.

Reply:
I'm not exactly sure what you mean by that

Reply:
Eventually, I'd imagine that we would propagate C attributes like `__attribute__((__malloc__))` onto the transpiled Rust, which gives us more information about the meaning of these functions.  Any other `__attribute__((__malloc__))` function should be treated like `malloc`.  This is not really important at the moment, though, but I just wanted to point it out for the eventual future.